helpers/password.go

```diff
@@ -6,3 +6,9 @@ func HashPassword(password string) (string, error) {
 	bytes, err := bcrypt.GenerateFromPassword([]byte(password), 14)
 	return string(bytes), err
 }
+
+func CheckPasswordHash(password string, hash string) (error) {
```

Comment:
How renaming it using `verify`?

```suggestion
func VerifyPasswordHash(password string, hash string) (error) {
```